Sources/Extensions/Optional.swift

```diff
@@ -1,30 +1,30 @@
 import Foundation
 
-public extension Swift.Optional {
+public extension Optional {
     public func then(f: (Wrapped) -> Void) {
         if let wrapped = self { f(wrapped) }
     }
 }
 
-// Thanks to John Sundell
-// https://github.com/JohnSundell/Require/
-public extension Swift.Optional {
-    /**
-     *  Require this optional to contain a non-nil value
-     *
-     *  This method will either return the value that this optional contains, or trigger
-     *  a `preconditionFailure` with an error message containing debug information.
-     *
-     *  - parameter hint: Optionally pass a hint that will get included in any error
-     *                    message generated in case nil was found.
-     *
-     *  - return: The value this optional contains.
-     */
+// Credits to John Sundell https://github.com/JohnSundell/Require/
```

Comment:
💄Splitting this into two lines is more readable:

```
// Credits to John Sundell 
// https://github.com/JohnSundell/Require/
```

Reply:
changed it for consistency with other places, but can change to two lines 💅